micall/core/plot_contigs.py

```diff
@@ -506,6 +511,9 @@ def build_contig(reader,
 
         arrows = []
         for arrow_start, arrow_end, blast_num in blast_ranges:
+            if arrow_end is None:
+                # Special case where an alignment ends with an insertion: ignore that insertion
+                continue
```

Comment:
Should you backtrack to the last position before the insert?

Reply:
Yes - it should be fixed now.